tendrl/commons/objects/definition/master.yaml

```diff
@@ -21,6 +21,17 @@ namespace.tendrl:
     ceph-provisioner: "provisioner/ceph"
     ceph-installer: "provisioner/ceph"
   flows:
+    ConfigureMonitoring:
```

Comment:
I have removed the "inputs" part as I need no "inputs" for this flow.
Is it ok?

Reply:
Assuming ConfigureMonitoring atom doesnt require any inputs,You still need whatever inputs that required by below atom

https://github.com/Tendrl/commons/blob/master/tendrl/commons/objects/cluster/atoms/check_cluster_available/__init__.py

Reply:
Ack